uobtheatre/bookings/exceptions.py

```diff
@@ -0,0 +1,21 @@
+from uobtheatre.utils.exceptions import GQLException
+
+
+class BookingTransferPerformanceUnchangedException(GQLException):
```

Comment:
We seem to be defining custom exceptions with two conventions. This method here doesn't allow overriding the message unlike https://github.com/BristolSTA/uobtheatre-api/pull/424/files#diff-bf8e7028d62bd485d9b2085357284db82dfbd2246bf3f727155969a40ed2df25R4. Might want to pick one to be consistent (I would favour the configurable message varient)